Emby.Server.Implementations/HttpServer/WebSocketConnection.cs

```diff
@@ -150,8 +150,8 @@ public async Task ProcessAsync(CancellationToken cancellationToken = default)
                 {
                     await ProcessInternal(pipe.Reader).ConfigureAwait(false);
                 }
-            } while (
-                (_socket.State == WebSocketState.Open || _socket.State == WebSocketState.Connecting)
+            }
+            while ((_socket.State == WebSocketState.Open || _socket.State == WebSocketState.Connecting)
```

Comment:
This one I dislike

Reply:
The analyzer warning, or just the specific way I changed it?

Reply:
The warning, imo the while should be on the same line